<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CreateNewMacro">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new macro </string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunExistingMacro">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Run existing macro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenSettings">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>open settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="BackgroundImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/ninja background smaller.PNG</pixmap>
    </property>
   </widget>
   <zorder>BackgroundImage</zorder>
   <zorder>CreateNewMacro</zorder>
   <zorder>RunExistingMacro</zorder>
   <zorder>OpenSettings</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../NEA/GUI resources.qrc"/>
 </resources>
 <connections/>
</ui>
